crates/orderbook/src/orderbook.rs

```diff
@@ -68,20 +67,42 @@ impl Metrics {
             .expect("unexpected error getting metrics instance")
     }
 
-    fn on_order_operation(order: &Order, operation: OrderOperation) {
-        let class = order_class_label(&order.metadata.class);
-        let op = operation_label(&operation);
-        Self::get().orders.with_label_values(&[class, op]).inc();
+    fn on_order_operation(order: &Order, operation: OrderOperation, quote: Option<&Quote>) {
+        let class = match quote {
+            Some(quote) if
+            // Check if the order at the submission time was billable (`user` order)
+                is_order_outside_market_price(
```

Comment:
This needs to be inverted, no?
Otherwise it would map an order with an out-of-market price to user orders if I'm not mistaken.
```suggestion
                !is_order_outside_market_price(
```

Reply:
Good catch, thanks!